pkg/apis/kubermatic/v1/constraint.go

```diff
@@ -120,5 +121,6 @@ type ConstraintList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
 
+	// Items refers to the list of Constraints that specifies a kubermatic wrapper for the gatekeeper constraints.
```

Comment:
If anything, it's "KKP", not "kubermatic". Also I don't think anyone cares that this is a wrapper. So maybe

> Items is a list of Gatekeeper Constraints